src/tree/expression/mod.rs

```diff
@@ -68,7 +68,7 @@ pub enum Expression {
     StringOperation(StringOperationExpression),
     TypeOperation(TypeOperationExpression),
     TernaryOperation(TernaryOperationExpression),
-    Identifier(Identifier),
+    Constant(Identifier),
```

Comment:
actually, left's revert this back, i remembered that this is also used to refer to class names in `Foo::bar()` so it's not only constants.